docs/docs/examples/vector_stores/DatabricksVectorSearchDemo.ipynb

```diff
@@ -192,6 +204,99 @@
     "\n",
     "print(response.response)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Advanced Usage (Auto Merging Retriever)\n",
+    "\n",
+    "This feature is enabled by the addition of node relationships to the index.\n",
```

Comment:
Can you clarify why users would use this? A simple explanation of the feature and why it's beneficial would be nice

Reply:
Also this appears to be a bit misleading - this sounds like AutoMergingRetriever is a feature of the DatabricksVectorSearch which is automatically enabled wen the index contains node relationship. Can we rephrase this to clarify (1) this is feature of `AutoMergingRetriever` class (2) setting node relationship is prerequisite.

Reply:
Added a bit more context and clarified how advanced features were merely due to the addition of node relationships in the index